util/kubectl.go

```diff
@@ -0,0 +1,118 @@
+package util
+
+// TODO: not sure how the Kubectl wrapper functions deal with whitespace in arguments.
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"syscall"
+
+	"github.com/fatih/color"
+)
+
+const (
+	// name of the kubectl executable
+	binaryName string = "kubectl"
+
+	// url to intallatzion instructions
+	kubectlInstallURL string = "http://kubernetes.io/docs/user-guide/prereqs/"
+
+	kubectlWindowsInstallURL string = "https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md"
+)
+
+// CheckKubectl checks if kubectl exists, is silent if yes, reports error if not
```

Comment:
I'd make this a little more precise - it _prints_ an error if kubectl doesn't exist, and also exits